src/Symfony/Bridge/Twig/Extension/FormExtension.php

```diff
@@ -166,6 +167,20 @@ public function isRootForm(FormView $formView)
         return null === $formView->parent;
     }
 
+    /**
+     * @internal
+     */
+    public function encodeCurrency(Environment $environment, $text)
+    {
+        if ('UTF-8' === $charset = $environment->getCharset()) {
+            return htmlspecialchars($text, ENT_QUOTES | (defined('ENT_SUBSTITUTE') ? ENT_SUBSTITUTE : 0), 'UTF-8');
```

Comment:
please use `\defined`, so that OPCache can resolve this at compile-time for recent PHP versions as it will know it is defined.